db/system_distributed_keyspace.cc

```diff
@@ -54,9 +61,37 @@ schema_ptr view_build_status() {
     return schema;
 }
 
+schema_ptr cdc_topology_description() {
+    thread_local auto schema = [] {
+        auto id = generate_legacy_id(system_distributed_keyspace::NAME, system_distributed_keyspace::CDC_TOPOLOGY_DESCRIPTION);
+        return schema_builder(system_distributed_keyspace::NAME, system_distributed_keyspace::CDC_TOPOLOGY_DESCRIPTION, {id})
+                .with_column("time", timestamp_type, column_kind::partition_key)
+                .with_column("description", bytes_type)
```

Comment:
Shouldn't this be text?

Reply:
Why? It's cdc::topology_description serialized.

Reply:
Then add a comment to make it clear.

Reply:
Or better, use the schema to describe the data using clustering keys and/or collections and/or UDTs. This will make it much easier to debug in the field, since you won't need to decode the blob.

Reply:
> Or better, use the schema to describe the data using clustering keys and/or collections and/or UDTs. This will make it much easier to debug in the field, since you won't need to decode the blob.

Thought about on how to do that, and arrived at the conclusion that it's not worth it.
What would be the representation of a sequence of (token, vector of streams, uint8_t), sorted by tokens?
- list? nay
- map of token -> (vector of streams, uint8_t)? yay

But then how to represent the token? Can't use string nor blob, because it won't sort correctly. Would have to use long_type, because I know we'll soon have all tokens as int64s, but we don't have that yet... Well I could use string/blob and then resort on read - meh. Also, converting between token and string/blob requires a partitioner instance - even more meh.

@haaawk provided some other arguments for using a serialized blob:
- if we need to extend this `topology_description` somehow, it'll be easier with a serialized value (although I don't believe we'll have to extend it)
- we'll need this serialization code anyway (in the future we won't use a table).

In summary, I'll just add a comment.

Reply:
Added a comment. See also docs/cdc.md

Reply:
The disadvantages of a serialized blob is that if we have to debug something in the field, the effort to decode the blob will be similar to the effort to write the bloc->schema conversion. And if we have to debug it twice, we start losing.

We can use bigint for the token and refuse to start cdc with an incompatible token type.

Extending the structure can be done by adding columns.